iOS/StitchCore/StitchCore/Core/Internal/StitchAppClientImpl.swift

```diff
@@ -16,6 +16,11 @@ internal final class StitchAppClientImpl: StitchAppClient {
         return _auth
     }
 
+    /**
+     * The client's underlying push notification component.
+     */
+    public var push: StitchPush
```

Comment:
[q][minor?] This doesn't seem correct. Why are we placing push in the StitchCore module?

Reply:
per offline discussion, leaving as is